src/data/stig_multipliers.ts

```diff
@@ -5,7 +5,7 @@ const SUPPORT_STIG_MULTIPLIERS = Object.freeze({
     T: [
       new Multiplier({
         active: true,
-        name: "Aslaug T: team fire DMG boost against Feathered enemies",
+        name: "Aslaug T: team ele DMG boost against Feathered enemies",
```

Comment:
nit: let's use the full word "elemental", would you mind changing it for the DPS stig multipliers too?